index.bs

```diff
@@ -754,13 +764,16 @@ MUST perform the following steps for each script execution environment:
      |allowedDevice| be the result.
   3. If |allowedDevice| is <code>null</code>, abort these steps.
   4. Let |deviceObj| be the {{USBDevice}} object representing |device|.
-  5. [=Fire an event=] named <dfn>connect</dfn> on {{Navigator/usb}}, using 
+  5. [=Fire an event=] named <dfn>connect</dfn> on |device|'s [=relevant global
+     object=]'s {{Navigator}} object's {{Navigator/usb}}, using
      {{USBConnectionEvent}}, with the <code>device</code> attribute set to 
      |deviceObj|.
 
 When the UA detects a <a>USB device</a> |device| has been disconnected from the
 host it MUST perform the following steps for each script execution environment:
 
+  1. If |device|'s [=relevant global object=] is not a {{Window}}, abort these
+     steps.
```

Comment:
Remove this line since you've added the note above about the worker needing to be active.

Reply:
Done